### ISSUE TITLE
Why Discriminative PT models with memory buffer degrade performance in table 1?

### ISSUE THREAD

**Issue by GoJunHyeong** [DoubleClass/GMM#10]
Hi, Thank you for your insightful paper!

I have some questions about Table 1.

You report PT models with a memory buffer, but there is some performance degradation when using exemplars.

However, the official paper reports performance enhancement when adapting exemplars.

I'm curious about this discrepancy. Could you clarify this part?

Thanks.
Regard.

**Comment by DoubleClass** [comment]
This is due to the prompt-based PT method, which lacks special design for the memory buffer. Taking L2P as an example, their classifier is entirely geared towards an exemplar-free setting. In the code, the logits related to old tasks are directly set to zero. Consequently, for exemplar data, the logits for the old classes are all zero, retaining only the logits for the new classes. This inevitably leads to incorrect predictions, thereby causing an overall performance decline.
